### PR TITLE
Add give popularity button to postbit

### LINKED ISSUE #8: Implement Quick Rep (Threads)
Find the javascript to trigger the reputation popup to give reputation.

Implement as Author or Postbit button.

### LINKED ISSUE #85: Give Popularity Postbit Button
Add a button to quickly rate a user next to the popularity number

### PR DESCRIPTION
**Pull request by moodiest**
Closes #85 
Closes #8

### CODE REVIEW

File: src/features/threads/GivePopularityPostbitButton.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const Feature = require("../../core/Feature");
+const Threads = require("../../sections/Threads");
+
+class GivePopularityPostbitButton extends Feature {
```
Comment (xadamxk) [inline src/features/threads/GivePopularityPostbitButton.js:4]: `GivePopularityPostbitButton `
to
`GivePopularityButton`

File: src/features/threads/GivePopularityPostbitButton.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        profile: "https://hackforums.net/member.php?action=profile&uid=2774521"
+      }
+    });
+  }
+
+  run() {
+    document.querySelectorAll("div.post_wrapper a[href*='reputation.php?uid=']").forEach((postbit) => {
+      let rate = document.createElement("a");
+
+      rate.href = `javascript:MyBB.reputation(${postbit.getAttribute("href").split("=")[1]})`;
```
Comment (xadamxk) [inline src/features/threads/GivePopularityPostbitButton.js:22]: This was really smart - good find!

**Review by xadamxk — CHANGES_REQUESTED** [review]
Made 1 change request.

Additionally, can you check what happens if a user hide's the reputation/popularity postbit item and tries to use this feature? If it errors out, could you add appropriate logic so it doesn't?

Thanks and good work on this!

**Comment by moodiest** [comment]
> Made 1 change request.
> 
> Additionally, can you check what happens if a user hide's the reputation/popularity postbit item and tries to use this feature? If it errors out, could you add appropriate logic so it doesn't?
> 
> Thanks and good work on this!

works fine as the display is just none so it still adds it to the element as it would normally, you just don't see it.

**Review by xadamxk — APPROVED** [review]

**Pull request merged**